package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "rhea",
-    "version": "1.0.6",
+    "version": "1.0.7",
```

Comment:
Changing an enum to string literal is a breaking change. However as pointed out in #224, that the enum type definition was not usable anyways. Hence bumping the patch version should be fine.

@grs - Please publish the new version of rhea, when possible. Thank you :).